src/sync/transforms/branch.ts

```diff
@@ -95,11 +101,8 @@ export const transformBranches = async (payload) => {
 	}
 
 	return {
+		... transformRepositoryDevInfoBulk(payload.repository, gitHubBaseUrl),
```

Comment:
Drag-n-drop replacement (with exception of changing `repository.name` to `repository.full_name` - this is what we have in other places), not feature-flagging
<img width="1432" alt="image" src="https://user-images.githubusercontent.com/20631664/194985472-4fa5b29e-71d5-4b2f-b3d6-50fb8a8d7e0c.png">